indra/newview/llagentcamera.cpp

```diff
@@ -937,6 +940,18 @@ void LLAgentCamera::cameraOrbitOver(const F32 angle)
     }
 }
 
+//-----------------------------------------------------------------------------
+// cameraRollOver()
+//-----------------------------------------------------------------------------
+void LLAgentCamera::cameraRollOver(const F32 angle)
+{
+    mRollAngle += fmod(angle, F_TWO_PI);
```

Comment:
Should use fmodf instead to avoid type conversions
```suggestion
    mRollAngle += fmodf(angle, F_TWO_PI);
```